backend/src/routing/handlers.rs

```diff
@@ -87,3 +87,29 @@ pub async fn deployments(
         get_deployments(&state.env_vars, &auth.username).await?,
     ))
 }
+
+#[derive(Deserialize)]
+/// The request format for the get environment variables endpoint
+pub struct EnvVarsReq {
+    project_name: String,
+}
+
+/// Gets the environment variables for a project if the user has access to it
+pub async fn get_env_vars(
+    State(state): HandlerState,
+    Extension(auth): Extension<Auth>,
+    Json(body): Json<EnvVarsReq>,
+) -> HandlerReturn<Vec<ProjectEnvVar>> {
+    let project_name = body.project_name.as_str();
```

Comment:
Does the frontend get the directory name? Or just the repo name? For eg, iqps-go is deployed in the `iqps` directory. We would ideally want the frontend to send `iqps` so that there is no need to search for a project.

Reply:
`project_name` is the directory name